spec/services/claim_fast_tracking/max_rating_annotator_spec.rb

```diff
@@ -13,7 +13,9 @@
       disabilities_data.map { |dis| DisabilityCompensation::ApiProvider::RatedDisability.new(**dis) }
     end
 
-    context 'when a disabilities response contains tinnitus and other disabilities' do
+    context 'with disability_526_maximum_rating disabled' do
```

Comment:
When the feature flag toggles between two different implementations, you'll want to update this spec to have an example for each implementation, and check that their behaviors are equivalent.